functions.php

```diff
@@ -39,6 +39,39 @@ function one_dashboard_setup() {
 		 */
 		add_theme_support( 'title-tag' );
 	}
+
+	/*
+	* Add column to posts table.
+	* @param array $column_name Name of the column.
```

Comment:
Add blanck space just before @param and not add blank line after param, you also forgot to add return statement type.